health/health/src/main/java/io/helidon/health/HealthSupport.java

```diff
@@ -260,9 +259,9 @@ public static HealthSupport create(Config config) {
      * Fluent API builder for {@link io.helidon.health.HealthSupport}.
      */
     public static final class Builder implements io.helidon.common.Builder<HealthSupport> {
-        private final Set<HealthCheck> allChecks = new LinkedHashSet<>();
-        private final Set<HealthCheck> livenessChecks = new LinkedHashSet<>();
-        private final Set<HealthCheck> readinessChecks = new LinkedHashSet<>();
+        private final List<HealthCheck> allChecks = new LinkedList<>();
```

Comment:
Just double-checking.  Repeatable semantics here are OK?

Reply:
It would be called twice. There was a problem that some of our health checks just disappeared (we ended up with 2 out of 6 in a test). Some quirk most likely caused by Weld proxies and their equals/hashCode methods.

Reply:
I did consider using an identity set to at least prevent the same instances. Yet in MP, we register this from selected beans (so it should be unique), and in SE the user registers checks by hand (so they have full control).